python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -16,6 +16,7 @@ class DagsterEcsTaskDefinitionConfig(
             ("container_name", str),
             ("command", Optional[Sequence[str]]),
             ("log_configuration", Optional[Mapping[str, Any]]),
+            ("repository_credentials", Optional[str]),
```

Comment:
These changes will allow you to set the repository credentials at the instance level (for example, in your dagster.yaml). That means every code location will receive the same repository credentials.

If you also thread these through to the "container context," you'll be able to see these at a code location level as well. [This PR is probably our best documentation at the moment for how to make such a change.](https://github.com/dagster-io/dagster/pull/14608)